metron-deployment/roles/ambari_config/vars/small_cluster.yml

```diff
@@ -87,6 +87,8 @@ configurations:
       topology.classpath: '{{ topology_classpath }}'
   - kafka-broker:
       log.dirs: '{{ kafka_log_dirs | default("/kafka-log") }}'
```

Comment:
Should the EC2 Readme include documentation about the dev env and what it means?  Should it link to or document how to .... change from dev to 'other', or do other from the start?

Reply:
Just pushed a change. How's that @ottobackwards?

Reply:
perfect, thank you @mmiklavc